evaluation/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+from transformers import AutoModelForCausalLM
+
+def load_model(model_name_or_path):
+    return AutoModelForCausalLM.from_pretrained(model_name_or_path)
```

Comment:
I think we should move this to a separate file and leave `__init__.py` blank

Reply:
Yup, totally agree. This was hasty code written on the fly. Fixed now!